extension/scripts/features/abroadpeoplefilter/ttPeopleFilter.js

```diff
@@ -0,0 +1,311 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"People Filter",
+		"travel",
+		() => settings.pages.travel.peopleFilter,
+		initialiseFilters,
+		addFilters,
+		removeFilters,
+		{
+			storage: ["settings.pages.travel.peopleFilter"],
+		},
+		null
+	);
+
+	function initialiseFilters() {
+		CUSTOM_LISTENERS[EVENT_CHANNELS.PEOPLE_SWITCH_PAGE].push(filtering);
+	}
+
+	async function addFilters() {
+		await requireElement(".users-list");
+
+		const { content } = createContainer("People Filter", {
+			nextElement: document.find(".users-list-title"),
+		});
+		content.innerHTML = `
+			<div class="filter-header">
+				<div class="statistic" id="showing">Showing <span class="filter-count">X</span> of <span class="filter-total">${
+					document.findAll(".users-list > *").length
+				}</span> users</div>
+			</div>
+			<div class="filter-content">
+				<div class="filter-wrap" id="activity-filter">
+					<div class="filter-heading">Activity</div>
+					<div class="filter-multi-wrap">
+						<div class="tt-checkbox-wrap">
+							<input type="checkbox" value="online" id="tt-people-filter-online">
+							<label for="tt-people-filter-online">Online</label>
+						</div>
+						<div class="tt-checkbox-wrap">
+							<input type="checkbox" value="idle" id="tt-people-filter-idle">
+							<label for="tt-people-filter-idle">Idle</label>
+						</div>
+						<div class="tt-checkbox-wrap">
+							<input type="checkbox" value="offline" id="tt-people-filter-offline">
+							<label for="tt-people-filter-offline">Offline</label>
+						</div>
+					</div>
+				</div>
+				<div class="filter-wrap">
+					<div class="filter-heading">Faction</div>
+					<div class="filter-multi-wrap">
+						<select name="faction" id="tt-faction-filter">
+							<option selected value="">none</option>
+							<option disabled value="------">------</option>
+						</select>
+					</div>
+				</div>
+				<div class='filter-wrap'>
+					<div class='filter-heading'>Special</div>
+					<div class="filter-multi-wrap">
+						<div class='tt-checkbox-wrap'>Y:<input type='checkbox' value='newplayer-yes'>N:<input type='checkbox' value='newplayer-no'>New Player</div>
```

Comment:
It would be great if the 'Y' and the 'X' are in a label.

Also counts for the ones beneath here.